spec/functions/certname_spec.rb

```diff
@@ -9,6 +9,5 @@
     ['test-vm.puppet.vm']
   end
 
-  # TODO: this *should* work, but is failing in TravisCI. Not sure why. It works on my laptop...
-  xit { in_bolt_context { is_expected.to run.with_params(target).and_return('test-vm.puppet.vm') } }
+  it { in_bolt_context { is_expected.to run.with_params(target).and_return('test-vm.puppet.vm') } }
```

Comment:
It seems `in_bolt_context` doesn't work well in this case.. It works for me in the following way though.. Insert the code below after the line 6 (`include BoltSpec::BoltContext`):
```ruby
  around(:each) do |example|
    in_bolt_context do
      example.run
    end
  end
```
and remove the `in_bolt_context {}` from the example. Then it should be fine. I have no idea what is the difference though as it should do the same.. maybe some Ruby guru will enlight us here

Reply:
Thanks a lot!